### PR TITLE
KG - Default Participant Arm

### PR DESCRIPTION
**Pull request by kyle-glick**
2 Scenarios:
Protocol has 1 arm: New Participants are automatically assigned to that arm

Protocol has 0 or multiple arms: New Participants are not assigned an arm

### CODE REVIEW

File: spec/support/ajax.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def wait_for_ajax
   end
 
   def finished_all_ajax_requests?
-    page.evaluate_script('jQuery.active').zero?
+    page.evaluate_script('jQuery.active') == 0
```
Comment (wtholt) [inline spec/support/ajax.rb]: .zero? deprecated?

File: spec/controllers/participants_controller_spec.rb [hunk 2]
```diff
@@ -41,6 +42,38 @@
         }
       }.to change(Participant, :count).by(1)
     end
+
+    it "should assign the arm if there is only one arm on a protocol" do
+      attributes = @participant.attributes
```
Comment (wtholt) [inline spec/controllers/participants_controller_spec.rb]: If you just want an attributes hash:

      create(:participant, ...) -> attributes_for(:participant, ...)

File: app/helpers/participant_helper.rb [hunk 3]
```diff
@@ -88,4 +89,10 @@ def participant_report_formatter participant
     ul        = raw content_tag(:ul, '', class: 'document-dropdown-menu hidden', id: "document_menu_participant_report_#{participant.id.to_s}", role: 'menu')
     html      = raw content_tag(:div, button + ul, class: 'btn-group')
   end
+
+  def change_arm_select participant
```
Comment (wtholt) [inline app/helpers/participant_helper.rb]: def change_arm_select(participant)

**Comment by kyle-glick** [comment]
@wtholt 
There has been an issue we've run in to in the past with .zero? where it is being called on a null value which does not have the .zero? method, so instead, we changed it to the zero evaluation.

**Pull request closed without merging**